wesnoth-utbs.po

```diff
@@ -1644,19 +1644,14 @@ msgstr ""
 
 #. [message]: speaker=Vengeful Lord
 #: data/campaigns/Under_the_Burning_Suns/scenarios/02_Across_the_Harsh_Sands.cfg:1223
-#, fuzzy
-#| msgid ""
-#| "Elf with a bow, I remember you. You thought me defeated, but I am back "
-#| "even more powerful than before. Death reigns eternal, your scorched bones "
-#| "and lifeless body will join my host."
 msgid ""
 "Elf of the sands, I remember you. You thought me defeated, but I am back "
 "even more powerful than before. Death reigns eternal, your scorched bones "
 "and lifeless body will join my host."
 msgstr ""
-"Elfe, ty s tím lukem, pamatuji si tě. Mysleli jste si, že jsem poražen, ale "
-"vracím se ve formě ještě silnější než dříve. Smrt vládne navěky, a brzy mi "
-"budou sloužit i vaše dobité kosti a bezduchá těla!"
+"Pamatuji si tě, elfe z pouště. Mysleli jste si, že jsem poražen, ale vracím "
+"se ještě v silnější formě než dříve. Smrt vládne navěky. Brzy mi budou "
+"sloužit i vaše dobité kosti a bezduchá těla!"
```

Comment:
Co myslíte, není to o trochu hladší čtení, bez toho měkkého **i** po **b**? Jen se domýšlím, co tím chtěl původní překladatel říci.
```suggestion
"sloužit i vaše sežehlé kosti a bezduchá těla!"
```